Python/getargs.c

```diff
@@ -1705,8 +1705,10 @@ vgetargskeywords(PyObject *args, PyObject *kwargs, const char *format,
             }
             if (max < nargs) {
                 PyErr_Format(PyExc_TypeError,
-                             "Function takes %s %d positional arguments"
+                             "%.200s%s takes %s %d positional arguments"
                              " (%d given)",
+                             (fname == NULL) ? "function" : fname,
```

Comment:
Any reason why you changed from uppercase Function to lowercase “function”?

It may be worth changing an exception message if you can make it less confusing or easier to read. But changing just the letter case does not seem worth the pain of breaking the old message.